src/components/_card.scss

```
@@ -78,3 +78,255 @@ a.#{$ns}card:focus,
+
+$sb-thumbnail-image-list: mood nps opentext responses;
+$sb-thumbnail-count-map: ('xs': 3, 's': 4, 'm': 5, 'l': 6);
+$sb-thumbnail-breakpoint-map: ('xs': 690px, 's': 960px, 'm': 1080px, 'l': 1280px);
+$sb-thumbnail-breakpoint-list: xs s m l;
+$sb-thumbnail-container-padding: 'xl';
+$sb-thumbnail-margin: 's';
+
+.#{$ns}thumbnail-container {
+  display: flex;
+  margin: 0 auto;
+  padding: 0 sb-rem-spacing($sb-thumbnail-container-padding);
+}
+
+.#{$ns}card-wrap {
+  margin: 0 -#{sb-rem-spacing($sb-thumbnail-margin)};
+}
+
+.#{$ns}card--compact {
+  box-shadow: $sb-card-compact-shadow;
+  cursor: pointer;
+  height: sb-px2rems($sb-card-compact-height);
+  overflow: hidden;
+  width: sb-px2rems($sb-card-compact-width);
+
+  .#{$ns}card__header {
+    align-items: unset;
+    justify-content: unset;
+    padding: 0;
+  }
+
+  .#{$ns}card__body {
+    border: none;
+    padding: sb-rem-spacing('m');
+    text-align: center;
+  }
+
+  &:hover,
+  &:focus,
+  &:focus-within {
+    border-color: $sb-color-blue-2;
+    box-shadow: $sb-card-compact-active-shadow;
+    transform: translateY(-3px);
+
+    .#{$ns}thumbnail__title {
+      color: $sb-thumbnail-title-active-color;
+    }
+  }
+}
+
+.#{$ns}thumbnail__wrapper {
+  display: flex;
+  flex: 1;
+  margin: 0;
+}
+
+.#{$ns}thumbnail__image {
+  background-size: cover;
+  background-position: bottom center;
+  display: block;
+  height: auto;
+  width: 100%;
+}
+
+@each $image in $sb-thumbnail-image-list {
+  .#{$ns}thumbnail__image--#{$image} {
+    background-image: url('#{$sb-assets-path}/media/thumbnail_#{$image}.png');
+  }
+}
+
+.#{$ns}thumbnail__title {
+  color: $sb-thumbnail-title-color;
+  font-family: $sb-primary-font-family;
+  font-size: sb-px2rems(16px);
+  font-weight: 500;
+  line-height: sb-px2rems(19px);
+  transition: color $sb-transition;
+}
+
+.#{$ns}thumbnail__description {
+  color: $sb-thumbnail-desc-color;
+  font-family: $sb-primary-font-family;
+  font-size: sb-px2rems(13px);
+  line-height: sb-px2rems(16px);
+}
+
+@each $sb-thumbnail-breakpoint in $sb-thumbnail-breakpoint-list {
+  $thumbnail-padding-size: 2 * map-get($sb-spacing-map, $sb-thumbnail-container-padding);
+  $thumbnail-margin:  2 * map-get($sb-spacing-map, $sb-thumbnail-margin);
+  $thumbnail-count: map-get($sb-thumbnail-count-map, $sb-thumbnail-breakpoint);
+  $breakpoint: map-get($sb-thumbnail-breakpoint-map, $sb-thumbnail-breakpoint);
+  $thumbnail-total-width: ($thumbnail-count * $sb-card-compact-width) + ($thumbnail-count * $thumbnail-margin) + $thumbnail-padding-size + 1;
+  $thumbnail-container-width: $thumbnail-total-width - $thumbnail-margin;
+
+  @media only screen and (min-width: $breakpoint) {
+    .#{$ns}thumbnail-container {
+      max-width: $thumbnail-container-width;
+    }
+  }
+}
+
+// Card - compact with radio button selection
+//
+// Browser support: IE11+, Safari6.1+, Firefox30+, Chrome30+
+//
+// Markup:
+// <div class="sb-thumbnail-container">
```

I think we are missing some aria labels and role in this example.
```suggestion
// <div class="sb-thumbnail-container" role="radiogroup">
```

---

Might even want to change it to a `fieldset` in this particular case actually (which would mean we'd need to reset the browser-default margins/paddings as well).

---

since we are not actually creating a `form` element, so I think `fieldset` wont be necessary. what do you think?

---

Hmm, dunno - it's a radio group. Not 100% sure on on the semantics of `fieldset`, if it should only be inside a `form` or not.